docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md

```diff
@@ -0,0 +1,158 @@
+1. In the toolbar, click the filter button.
+
+2. In the **Filters** pane, click **+ ADD** and select **store**.
+  This adds the filter to the pane.
+3. To edit the filter, click on it.
+4. To apply the filter to all the visual in the sheet, for **Applied to**, select **Single sheet**.
+5. To add filter controls, click the three dots next to the filter and select **Add control**>**Top of this sheet**.
+  This adds the filter controls to the top of the sheet.
+
+![store-filter](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/store-filter.png)  
+
+
+## Share the analysis
+
+If you want another author to collaborate with you on the analysis, you can share it.
+
+To share an analysis, follow the steps:
+
+1. On the page of the analysis, go to **File** > **Share**.
+2. In the **Share analysis**, enter and select the email address of the author to share the analysis with.
+  This displays the user in the table.
+3. Repeat the previous step until you've added all the authors you want to share the analysis with.
+4. Click **Share**
+  This closes the window. The added users can now edit the analysis.
+
+{% info_block infoBox "Manage permissions" %}
+
+You can check and manage permissions of all users to an analysis in **File**>**Share**>**Manage analysis permissions**.
+
+{% endinfo_block %}
+
+
+
+
+## Publish analysis as a dashboard and share
+
+Most of your analytics users will have view permissions. When you're ready to share the analysis with a wider audience, you need to publish it as a dashboard. Users with view permissions can access only dashboards.
+
+To publish and share a dashboard, follow the steps:
+
+1. To publish a dashboard, on the page of an analysis click **PUBLISH**.
+2. For **Publish new dashboard as**, enter a name for the dashboard.
+3. To confirm, click **Publish dashboard**.
+  This publisher the dashboard and opens its page.
+
+![published-dashboard](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/best-practices-analyzing-shop-data.md/published-dashboard.png)
+
+4. To share the dashboard, in the top-right corner, click the share icon and select **Share dashboard**.
+  This opens the **Share dashboard** page.
+5. In the **Invite users and groups to dashboard** pane, start entering the email address of a user.
+  This shows users with matching email addresses as you're typing.
+6. Next to the needed user, click **ADD** and select the permissions to assign to them.
+  This shows the user in the **Manage permissions** pane with the permissions you've assigned.
+
+
+## Conclusion
+
+In this guide, you've followed a typical user journey of creating, collaborating, and presenting data in a meaningful way. Now you can play around with these tools or learn by following [Author Workshop](https://catalog.workshops.aws/quicksight/en-US/author-workshop).
```

Comment:
let's please this workshop here as well please:
https://explore.skillbuilder.aws/learn/course/external/view/elearning/14908/getting-started-with-amazon-quicksight